File: Backend/DisasterDispatch.Core/Entities/NotificationUser.cs
using DisasterDispatch.Core.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Entities
{
    public class NotificationUser:BaseEntity
    {
        public Guid NotificationId{ get; set; }
        public Notification Notification{ get; set; }
        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }

    }
}

File: Frontend/DisasterDispatchFrontend/src/app/Models/notificationuser.ts
import { Guid } from "guid-typescript";
import { Notificationcl } from "./notificationcl";
import { AppUser } from "./app-user";

export class Notificationuser {
    notificationId:Guid
    notification:Notificationcl;
    appUserId:string;
    AppUser:AppUser
}
